plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -484,34 +476,4 @@ public DeltaLakeConfig setRegisterTableProcedureEnabled(boolean registerTablePro
         this.registerTableProcedureEnabled = registerTableProcedureEnabled;
         return this;
     }
-
-    @Min(value = MIN_READER_VERSION, message = "Must be in between " + MIN_READER_VERSION + " and " + MAX_READER_VERSION)
-    @Max(value = MAX_READER_VERSION, message = "Must be in between " + MIN_READER_VERSION + " and " + MAX_READER_VERSION)
-    public int getDefaultReaderVersion()
```

Comment:
[TestDeltaLakePageSink.java](https://github.com/trinodb/trino/pull/16165/files#diff-66264364c4b34e57284c11079858ad05f15bf7e5ad63b06f3d3e18646efaa4da) and [TestDeltaLakeConfig.java](https://github.com/trinodb/trino/pull/16208/files#diff-bba17fc0643cf731516d7bf56e5736dd1e6c8dad8aeb62a52fa1758d3acdb4d3) needs to be updated.

Reply:
[DeltaLakeMetadataFactory.java](https://github.com/trinodb/trino/pull/16208/files#diff-d50f059e35f4a166854e3c70c2a5ba77f3abaa9dbd1e7caad4aef1371b454e60) needs to be updated.